app/enterprise/2.1.x/brain-immunity/model-training

```diff
@@ -0,0 +1,119 @@
+---
+title: Immunity Model Training
+---
+
+### Introduction
+
+Immunity automatically starts training its models once it is up and running and receiving data. Immunity will create a unique model for every unique endpoint + method combination it sees in incoming traffic. For example, if you have an endpoint [www.test-website.com/buy](http://www.test-website.com/buy) and traffic comes in with both GET and POST requests for that endpoint, Immunity will create two models; one for the endpoint + GET traffic and one for the endpoint + POST traffic.
```

Comment:
Need some line wrapping (global comment). Suggest 80 char max. Also should not use a + sign, use "and" instead.